plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2903,6 +2904,31 @@ private boolean isGuestAarch64() {
         return AARCH64.equals(guestCpuArch);
     }
 
+    private boolean isQemuVersionConflictingSMBiosForWindowsGuestVMs() {
+        if (!isUbuntuOrDebianHost()) {
```

Comment:
we don't know if this issue is restricted to only ubuntu/debian - el10 "may" also face this issue..